src/routes/lifi/utils.ts

```diff
@@ -0,0 +1,173 @@
+import type { ChainId } from '@lifi/sdk';
+import {
+  getStatus,
+  type GetStatusRequest,
+  type StatusResponse,
+} from '@lifi/sdk';
+import type {
+  Chain,
+  ChainContext,
+  TokenId,
+  TransactionId,
+  Network,
+  Signer,
+} from '@wormhole-foundation/sdk-connect';
+import {
+  Wormhole,
+  chainToPlatform,
+  isSignAndSendSigner,
+  isSignOnlySigner,
+  isNative,
+  nativeChainIds,
+} from '@wormhole-foundation/sdk-connect';
+import type { EvmPlatform } from '@wormhole-foundation/sdk-evm';
+import type { SolanaPlatform } from '@wormhole-foundation/sdk-solana';
+import type { SuiPlatform } from '@wormhole-foundation/sdk-sui';
+import {
+  LIFI_NATIVE_ADDRESS_EVM,
+  LIFI_NATIVE_ADDRESS_SVM,
+  LIFI_NATIVE_ADDRESS_SUI,
+  CHAIN_ID_MAP,
+  CHAIN_FROM_ID_MAP,
+} from './consts';
+import { generateThrowawayAddress as generateEvmThrowawayAddress } from './platforms/evm';
```

Comment:
circular loop importing from there, and that imports from here

Reply:
changed